k8s-install/README.md

```diff
@@ -1,11 +1,18 @@
 # Kubernetes self-hosted install
 
-Installation is simple:
+Installation is simple, for Kubernetes 1.5:
 
 ```
 kubectl apply -f canal.yaml
 ```
 
+For Kubernetes 1.6:
+
+```
+kubectl apply -f canal.yaml
+kubectl apply -f rbac.yaml
```

Comment:
I think there's still an issue - the rbac.yaml is for the k8s datastore yaml, not this one.